<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1280</width>
    <height>720</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>HiTagget</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../../../../UPC/CICLO 3/Algoritmos y estructura de datos/Semana 7/logo_icon.ico</normaloff>../../../../UPC/CICLO 3/Algoritmos y estructura de datos/Semana 7/logo_icon.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(21, 32, 43);
font: 14pt &quot;Comic Sans MS&quot;;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListWidget" name="listWidgetCont">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>341</y>
      <width>321</width>
      <height>271</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Courier</family>
      <pointsize>8</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(28, 44, 58);
border-top-color: rgb(25, 39, 52);
font: 8pt &quot;Courier&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetPubli">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>79</y>
      <width>581</width>
      <height>491</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>581</width>
      <height>520</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(28, 44, 58);
border-top-color: rgb(25, 39, 52);
font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAsNeeded</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCamFoto">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>120</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;
background-color: rgb(128, 183, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Actualizar Foto</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblFPerfil">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>24</y>
      <width>111</width>
      <height>91</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>:/D:/UPC/CICLO 3/Algoritmos y estructura de datos/TP/Usuario.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="lblUsuario">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>151</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(35, 182, 111);
border-color: rgb(255, 255, 0);
background-color: rgb(28, 44, 58);
font: 13pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetFollowers">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>281</width>
      <height>251</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(35, 182, 111);
color: rgb(255, 255, 255);
background-color: rgb(28, 44, 58);
font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
   </widget>
   <widget class="QListWidget" name="listWidgetTend">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>40</y>
      <width>321</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(28, 44, 58);
border-top-color: rgb(25, 39, 52);
font: 9pt &quot;Arial&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>0</number>
    </property>
    <property name="sizeAdjustPolicy">
     <enum>QAbstractScrollArea::AdjustToContentsOnFirstShow</enum>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::NoSelection</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="btnPublicar">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>650</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;
background-color: rgb(128, 183, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Publicar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnAgregar">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>650</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;
background-color: rgb(128, 183, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Agregar</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>580</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Criterio:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Publicaciones:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>20</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Tendencias:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>305</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Contactos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>150</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="text">
     <string>Seguidos:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>186</x>
      <y>20</y>
      <width>119</width>
      <height>99</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(35, 182, 111);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="BCnom">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>650</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;
background-color: rgb(128, 183, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BCnum">
    <property name="geometry">
     <rect>
      <x>1120</x>
      <y>650</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;
background-color: rgb(128, 183, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Numero</string>
    </property>
   </widget>
   <widget class="QPushButton" name="BCapod">
    <property name="geometry">
     <rect>
      <x>1190</x>
      <y>650</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 10pt &quot;Arial&quot;;
background-color: rgb(128, 183, 128);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>Apodo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>620</y>
      <width>201</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Ordenar por:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cb_criterios">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>610</y>
      <width>171</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Interaciones</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Fecha</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Likes</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="txtSearchBox">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>39</y>
      <width>421</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(35, 182, 111);
font: 8pt &quot;Comic Sans MS&quot;;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cmbBoxSearchOptions">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>40</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Igual a</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Inicia con</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Finaliza con</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Contiene</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>No contiene</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="btnInvertir">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>580</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>14</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(128, 183, 128);
color: rgb(255, 255, 255);
font: 14pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="text">
     <string>Invertir</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblShownUser">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>470</y>
      <width>281</width>
      <height>101</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Comic Sans MS</family>
      <pointsize>13</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(35, 182, 111);
border-color: rgb(255, 255, 0);
background-color: rgb(28, 44, 58);
font: 13pt &quot;Comic Sans MS&quot;;</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <zorder>label_9</zorder>
   <zorder>listWidgetCont</zorder>
   <zorder>listWidgetPubli</zorder>
   <zorder>btnCamFoto</zorder>
   <zorder>lblFPerfil</zorder>
   <zorder>lblUsuario</zorder>
   <zorder>listWidgetFollowers</zorder>
   <zorder>listWidgetTend</zorder>
   <zorder>btnPublicar</zorder>
   <zorder>btnAgregar</zorder>
   <zorder>label_2</zorder>
   <zorder>label_5</zorder>
   <zorder>label_6</zorder>
   <zorder>label_7</zorder>
   <zorder>label_8</zorder>
   <zorder>BCnom</zorder>
   <zorder>BCnum</zorder>
   <zorder>BCapod</zorder>
   <zorder>label_10</zorder>
   <zorder>cb_criterios</zorder>
   <zorder>txtSearchBox</zorder>
   <zorder>cmbBoxSearchOptions</zorder>
   <zorder>btnInvertir</zorder>
   <zorder>lblShownUser</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1280</width>
     <height>37</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <tabstops>
  <tabstop>btnPublicar</tabstop>
  <tabstop>btnAgregar</tabstop>
  <tabstop>listWidgetPubli</tabstop>
  <tabstop>listWidgetTend</tabstop>
  <tabstop>listWidgetFollowers</tabstop>
  <tabstop>btnCamFoto</tabstop>
  <tabstop>listWidgetCont</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
